django/db/models/base.py

```diff
@@ -392,6 +392,11 @@ class ModelState:
 
 
 class Model(metaclass=ModelBase):
+    def __new__(_cls, *args, **kwargs):
```

Comment:
any explanation about the benefits of this change?

Reply:
Kindly see the ticket https://code.djangoproject.com/ticket/30083